x-pack/test/alerting_api_integration/spaces_only/tests/alerting/find.ts

```diff
@@ -171,6 +189,62 @@ const findTestUtils = (
         expect(response.body.total).to.equal(1);
         expect(response.body.data[0].params.strValue).to.eql('my b');
       });
+
+      it('should sort by parameters', async () => {
+        const response = await supertest.get(
+          `${getUrlPrefix(Spaces.space1.id)}/${
+            describeType === 'public' ? 'api' : 'internal'
+          }/alerting/rules/_find?sort_field=params.severity&sort_order=asc`
+        );
+        expect(response.body.data[0].params.severity).to.equal('low');
+        expect(response.body.data[1].params.severity).to.equal('medium');
+        expect(response.body.data[2].params.severity).to.equal('high');
```

Comment:
I'm not sure why but this test fails in CI, but I can't seem to get it to fail locally (always passes)

Reply:
How are you running it locally? Using `--grep`?

Reply:
correct, using the functional test runner and `--grep="find"`.

Reply:
Try running it a level up until you find the error, so instead of `--grep="find"`, try `--grep="Alerting"`

Reply:
Ah got it, makes sense, yea I'll try that